scatter.js

```diff
@@ -565,12 +565,14 @@ Scatter.prototype.update = function (...args) {
 				}
 
 				// update position buffers
-				positionBuffer({
-					data: f32.float(positions),
+				var float_data = f32.float32(positions)
+				state.positionBuffer({
+					data: float_data,
 					usage: 'dynamic'
 				})
-				positionFractBuffer({
-					data: f32.fract(positions),
+				var frac_data = f32.fract32(positions, float_data)
+				state.positionFractBuffer({
```

Comment:
Also here `state` is not defined!
Fixed in https://github.com/gl-vis/regl-scatter2d/commit/42113ead9fd2a2cf3f4c8043c71ee824dcba47c8